src/Lumina/Data/Files/Pcb/PcbResourceFile.cs

```
@@ -0,0 +1,197 @@
+                    while( reader.BaseStream.Position + header.HeaderSkip < finalPosition )
+                    {
+                        header.Children.Add( ResourceNode.ReadWithCount( reader, out var nodesRead ) );
+                        totalNodesRead += nodesRead;
+                    }
+
+                    reader.BaseStream.Position = finalPosition;
+                }
+
+                for( var i = 0; i < numVertFloat32; i++ )
+                {
+                    header.Vertices[ i ] = Common.Vector3.Read( reader );
+                }
+                
+                for( var i = numVertFloat32; i < numVertFloat32 + numVertFloat16; i++ )
+                {
+                    header.Vertices[ i ] = Common.Vector3.Read16( reader );
+                }
+
+                for( var i = 0; i < header.Polygons.Length; i++ )
+                {
+                    header.Polygons[i] = Polygon.Read( reader );
+                }
+                
+                
+                return header;
+            }
+        }
+        
+        public struct Polygon
+        {
+            public byte[] VertexIndex;
+            public ushort Unknown;
+            
+            public static Polygon Read( BinaryReader reader )
+            {
+                var polygon = new Polygon
+                {
+                    VertexIndex = reader.ReadBytes( 3 ),
+                };
+
+                // Padding
+                reader.BaseStream.Position += 2;
+
+                polygon.Unknown = reader.ReadUInt16();
+
+                // Padding
+                reader.BaseStream.Position += 5;
+
+                return polygon;
+            }
+        }
+
+        public PcbResourceHeader Nodes;
+
+        public override void LoadFile()
+        {
+            var streamStart = Reader.BaseStream.Position;
+            var isList = Reader.ReadUInt32() != 0;
+            Reader.BaseStream.Position = streamStart;
+
+            if( isList ) return;
```

i would probably throw an exception here instead because this is never a valid case